cleverhans/utils.py

```diff
@@ -113,6 +118,20 @@ def other_classes(nb_classes, class_ind):
     return other_classes_list
 
 
+def conv_2d(filters, kernel, strides, padding):
```

Comment:
Maybe `kernel_shape` instead of `kernel` to be clear the user isn't supposed to pass in a 4D tensor of parameters